develop/dev-guide-aws-quicksight-integration.md

```diff
@@ -0,0 +1,86 @@
+
+If you are interested in learning more about TiDB and ProxySQL, you can find some useful links as follows:
+
+- [TiDB Cloud](https://docs.pingcap.com/tidbcloud)
+- [TiDB Developer Guide](/develop/dev-guide-overview.md)
+- [Amazon QuickSight Documentation](https://docs.aws.amazon.com/quicksight/latest/user/welcome.html)
+
+## What is ProxySQL?
+
+[Amazon QuickSight](https://aws.amazon.com/quicksight/) is a cloud-scale business intelligence (BI) service that you can use to deliver easy-to-understand insights to the people who you work with, wherever they are. Amazon QuickSight connects to your data in the cloud and combines data from many different sources. In a single data dashboard, QuickSight can include AWS data, third-party data, big data, spreadsheet data, SaaS data, B2B data, and more. As a fully managed cloud-based service, Amazon QuickSight provides enterprise-grade security, global availability, and built-in redundancy. It also provides the user-management tools that you need to scale from 10 users to 10,000, all with no infrastructure to deploy or manage.
+
+## Why Amazon QuickSight integration?
+
+Every day, the people in your organization make decisions that affect your business. When they have the right information at the right time, they can make the choices that move your company in the right direction.
+
+Here are some of the benefits of using Amazon QuickSight for analytics, data visualization, and reporting:
+
+- The in-memory engine, called SPICE, responds with blazing speed.
+- No upfront costs for licenses and a low total cost of ownership (TCO).
+- Collaborative analytics with no need to install an application.
+- Combine a variety of data into one analysis.
+- Publish and share your analysis as a dashboard.
+- Control features available in a dashboard.
+- No need to manage granular database permissions—dashboard viewers can see only what you share.
+
+## Create TiDB Cluster
+
+TiDB (/’taɪdiːbi:/, "Ti" stands for Titanium) is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability.
+
+<CustomContent platform="tidb">
+
+If you don't have a TiDB cluster, you can follow the [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md) to create a TiDB cluster.
+
+</CustomContent>
+
+Here is the way to create a playground TiDB cluster. I’ll create a TiDB cluster using [TiDB Cloud Dedicated Tier](https://docs.pingcap.com/tidbcloud/select-cluster-tier#dedicated-tier). But you can create your TiDB cluster anywhere. Just ensure that Amazon QuickSight can connect to your TiDB cluster.
+
+You can follow the document to [Create a TiDB Cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster), to create a TiDB Cloud Dedicated Tier cluster.
+
+## Integrate Amazon QuickSight and TiDB
```

Comment:
We can use numbered list here to describe how to integrate step by step.

Reply:
Changed.